src/test/scala/com/amazon/deequ/SparkContextSpec.scala

```diff
@@ -80,11 +99,22 @@ trait SparkContextSpec {
       .config("spark.sql.shuffle.partitions", 2.toString)
       .config("spark.sql.adaptive.enabled", value = false)
       .config("spark.driver.bindAddress", "127.0.0.1")
+      .config("spark.sql.warehouse.dir", warehouseDir.toAbsolutePath.toString)
```

Comment:
Should this be added conditionally, where Iceberg support is required? This change affects all spark sessions used by all unit tests.

Reply:
It's not just for Iceberg, its used for other spark tables writes as well. I have updated to add this only when needed. 

If we do not set this when we run local tests, spark warehouse dir will get created in project root directory when executing unit tests.